updatecli/updatecli.d/jenkins-inbound-agent.yml

```
@@ -24,8 +24,8 @@ sources:
       username: "{{ .github.username }}"
       versionfilter:
         kind: regex
-        ## not the JDK8 version - regex from https://stackoverflow.com/questions/16398471/regex-for-string-not-ending-with-given-suffix/16398502#16398502
-        pattern: '.*[^-][^j][^d][^k][^8]$'
+        ## not the JDK7 version - regex from https://stackoverflow.com/questions/16397471/regex-for-string-not-ending-with-given-suffix/16397502#16397502
```

```suggestion
        ## not the JDK8 version - regex from https://stackoverflow.com/questions/16398471/regex-for-string-not-ending-with-given-suffix/16398502#16398502
```

Bad search and replace